### DISCUSSION TITLE
Procedure after each home assignments

### DISCUSSION THREAD

**Discussion by ansenpjames** [cafawo/FinancialDataAnalytics#3] [Q&A] [answered]
Could someone share the procedure ( how to upload in our GitHub repository- I hope this is the way we submit our home assignments ) after completing each week home assignments.

**Comment by birte0** [comment, marked as answer]
To submit your homework assignments, you'll first need to navigate to the cloned repository on your computer using the command line. Then, use the command line to add, commit, and push your solutions as explained in the "Commit and push" task at [https://github.com/cafawo/FinancialDataAnalytics/blob/master/homework/01_setup.ipynb](https://github.com/cafawo/FinancialDataAnalytics/blob/master/homework/01_setup.ipynb)

If that doesn't answer your question please let us know at which step you are facing difficulty.

**Reply by ansenpjames** [reply]
yeah..Thank you .. It worked
